src/tests/ftest/util/soak_utils.py

```diff
@@ -954,7 +956,7 @@ def create_macsio_cmdline(self, job_spec, pool, ppn, nodesperjob):
             macsio.timings_file_name.update(macsio_timing_log)
             env = macsio.env.copy()
             env["D_LOG_FILE"] = get_log_file(daos_log or "{}_daos.log".format(macsio.command))
-
+            env["DAOS_UNS_PREFIX"] = "daos://{}/{}/".format(macsio.daos_pool, macsio.daos_cont)
```

Comment:
FYI, there is a function
```suggestion
from duns_utils import format_path
            env["DAOS_UNS_PREFIX"] = format_path(macsio.daos_pool, macsio.daos_cont)
```